fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -2102,4 +2102,17 @@ public String getNotReadyReason() {
         }
         return "";
     }
+
+    public boolean exceedMaxJournalSize(BackupJob job) {
```

Comment:
It seems that we can change it to the following form
```suggestion
    public boolean exceedMaxJournalSize(Writable w) {
```